### PR TITLE
support storing external spice owner for vertices

### PR DESCRIPTION
**Pull request by sslash**

### CODE REVIEW

File: lib/fruits-lib.js [hunk 1]
```diff
@@ -6537,7 +6537,9 @@ return /******/ (function(modules) { // webpackBootstrap
 	            var req = _ref10.req;
 	            return req.put('/routes/' + routeId + '/vertices/' + verticeId + '/photo', {
 	                externalImage: url,
-	                photoSource: source
+	                photoSource: source,
+	                externalLink: externalLink,
```
Comment (skeie) [inline lib/fruits-lib.js:6541]: Bare: externalLink ? 
Samme med username

File: lib/fruits-lib.js [hunk 2]
```diff
@@ -6537,7 +6537,9 @@ return /******/ (function(modules) { // webpackBootstrap
...
+	                username: username
 	            });
 	        },
 	        meta: { id: verticeId, photo: { url: url, source: source } }
```
Comment (skeie) [inline lib/fruits-lib.js:6545]: her og? url, source ?

File: lib/fruits-lib.js (hunk 1 above)
Comment (sslash) [inline lib/fruits-lib.js:6541]: yeah

File: lib/fruits-lib.js (hunk 2 above)
Comment (sslash) [inline lib/fruits-lib.js:6545]: oui

**Pull request merged**